devtools/conda-recipe/meta.yaml

```diff
@@ -40,7 +40,7 @@ requirements:
     - openmm
     - parmed
     - openmoltools
-    - alchemy
+    - alchemy >=1.1.3
```

Comment:
`softcore_beta` was added in 1.2.0, so I'll update this:
https://github.com/choderalab/alchemy/releases